auth-web/src/components/auth/manage-business/AccountAuthorizationRequest.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div class="pa-5">
```

Comment:
I recommend adding an id to all components (or classname if there can be multiple instances of the component). Otherwise, looks good!